roofit/roofitcore/inc/RooFFTConvPdf.h

```diff
@@ -73,25 +73,23 @@ class RooFFTConvPdf : public RooAbsCachedPdf {
   void calcParams() ;
   Bool_t redirectServersHook(const RooAbsCollection& newServerList, Bool_t mustReplaceAll, Bool_t nameChange, Bool_t isRecursive) ;
 
-  Double_t*  scanPdf(RooRealVar& obs, RooAbsPdf& pdf, const RooDataHist& hist, const RooArgSet& slicePos, Int_t& N, Int_t& N2, Int_t& zeroBin, Double_t shift) const ;
+  std::vector<double>  scanPdf(RooRealVar& obs, RooAbsPdf& pdf, const RooDataHist& hist, const RooArgSet& slicePos, Int_t& N, Int_t& N2, Int_t& zeroBin, Double_t shift) const ;
```

Comment:
this is a breaking change if users were inheriting from RooFFTConvPdf and calling `scanPdf` in the derived class. is that a valid use of this class/method?

Reply:
There is no reason for a user to do this. Unfortunately RooFit uses `protected` too much for things that should actually be `private`.